_posts/2017-08-16-node_starter_guide.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Starters Guide to Node.js"
+date: 2017-08-16 00:00:00 +530
+author: Hritvik Taneja
+category:
+tags:
+- node
+- Node.js
+- npm
+---
+## `console.log('Hello World')`
+
+I assume that most of you are here to learn about Node.js. So, lets get started!!
+
+## What is Node.js ?
+
+**If you go by docs:** [Node.js®](https://nodejs.org/en/) is a JavaScript runtime built on [Chrome's V8 JavaScript engine](https://developers.google.com/v8/). Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient. Node.js' package ecosystem, [npm](https://www.npmjs.com/), is the largest ecosystem of open source libraries in the world.
+
+In simple words Node.js is server side programming environment which helps us to create APIs which can be consumed by Mobile devices, Websites etc..
+
+In 2009 Ryan Dahl released Node.js to run JavaScript on servers such as Linux and IIS Windows servers.
+
+## Why should one learn Node.js ?
+
+JavaScript is undoubtedly one of the most popular programming languages out there today, and for good reason. It can easily be run in your browser, on a server, on your desktop, or even on your phone as an app.
+
+Before Node.js, JavaScript was only client side language meaning it can run only on browsers such as chrome, Mozilla etc.
+
+You can find loads of stuff on the internet which will persuade you to learn Javascript and Node.js, but one reason which I find the most compelling is that you can pursue both frontend and backend development if you learn the above stated.
+
+## How to get started ?
+
+I assume that you know JavaScript but if you don't then you should probably first learn it, [this](http://www.amazon.in/Head-First-JavaScript-Programming-Morrison/dp/144934013X) is a very good book to start with.
+
+First, Node.js is not a programming language but a minimal framework. Language is Javascript. And it is very easy to learn.
+
+To begin with you should have [Node.js](https://nodejs.org/en/) installed on your machine(obviously), npm is installed along with it. There are a bunch of very good tutorials for Node.js available online. But I personally recommend to complete all the tutorials(exercises) of [learnyounode](https://www.npmjs.com/package/learnyounode), this might need a lot of googling and reading of [the docs](https://nodejs.org/api/). But if you are  not comfortable with the previously mentioned two thing(which you ideally should be if you want to become a good programmer) the go for [learn all the nodes](https://www.youtube.com/playlist?list=PLQmX5gaHg2SXyKuT9BQ_nbFIdZ39yeRxH) lecture series.
```

Comment:
Explain NPM a bit (more about the "largest ecosystem" thing could be said here).

Nits: 
* comma after "begin with"
* recommend to complete -> recommend completing
* thing( -> thing ( [added a space]